### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by NicolasMahe**
Add pull request template, same as liteflow-js

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!-- A concise description of what this PR is fixing or adding -->
+
+### How to test
+
+<!-- A concise explanation how to test this PR with links -->
+
+### Checklist
+
+- [ ] Update related changelogs <!-- Check [root's CHANGELOG.md](/CHANGELOG.md) to access the right changelogs -->
+- [ ] Update docs if necessary
```
Comment (cpvdeveloper) [inline .github/pull_request_template.md:19]: We don't really have docs for Starter Kit, do we? This might never get used

**Review by cpvdeveloper — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (antho1404) [reply .github/pull_request_template.md:19]: Yes the doc is actually here https://github.com/liteflow-labs/liteflow-js

**Review by ismailToyran — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (antho1404) [inline .github/pull_request_template.md:19]:
```suggestion
- [ ] Update docs if necessary <!-- Docs in https://github.com/liteflow-labs/liteflow-js/tree/main/docs/pages/starter-kit -->
```

Comment (antho1404) [reply .github/pull_request_template.md]: Still keeping it as it is still a good reminder to see that everytime we open a PR, just added a comment with the link to the source for the doc

**Pull request merged**